lib/token.h

```diff
@@ -1166,13 +1161,9 @@ class CPPCHECKLIB Token {
 
     const ValueFlow::Value * getInvalidValue(const Token *ftok, nonneg int argnr, const Settings *settings) const;
 
-    const ValueFlow::Value* getContainerSizeValue(const MathLib::bigint val) const;
-
     const Token *getValueTokenMaxStrLength() const;
     const Token *getValueTokenMinStrSize(const Settings *settings) const;
 
-    const Token *getValueTokenDeadPointer() const;
```

Comment:
Last usage removed in 1afd56e964781f71e862124fceca7a283d2a68d1 - leftover of the `deadpointer` check removal.